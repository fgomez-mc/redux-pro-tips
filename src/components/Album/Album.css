.Album {
	width: 100%;
	background-color:  silver;
	margin: 2px;
	padding: 10px;
	border: solid 1px gray;
	box-sizing: border-box;
}