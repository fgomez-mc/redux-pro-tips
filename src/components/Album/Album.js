import './Album.css';

export default function Album({ title="", id="" }) {
  return <h5 className='Album'>{id}) {title}</h5>;
}
