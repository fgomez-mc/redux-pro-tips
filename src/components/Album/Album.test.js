import renderer from "react-test-renderer";
import Album from "./Album";

describe("album", () => {
  it("match the snapshot", () => {
    const testRenderer = renderer.create(<Album />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
