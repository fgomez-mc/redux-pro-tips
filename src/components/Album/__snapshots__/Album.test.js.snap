// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`album match the snapshot 1`] = `
<h5
  className="Album"
>
  
  ) 
  
</h5>
`;
